client/app/lib/components/redux-form/Toggle.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import MaterialToggle from 'material-ui/Toggle';
+import createComponent from './createComponent';
+import mapError from './mapError';
+
+const errorStyle = {
+  color: 'red',
+};
+
+// Toggle implementation with an error displayed at the bottom.
+const Toggle = function ({ errorText, ...props }) {
+  return (
+    <div>
+      <MaterialToggle {...props} />
+      {
+        errorText &&
+        <div style={errorStyle}>{errorText}</div>
+      }
+    </div>
+  );
+};
+
+Toggle.propTypes = {
+  errorText: PropTypes.string,
+};
+
+export default createComponent(
```

Comment:
Ah ok - I meant the documentation for the toggle component, like how to use it and what props to pass in. But maybe there is no need to since it seems quite straightforward, what do you think?